torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -770,9 +809,18 @@ class MKLDNNSubgraphSlicer {
       // conversions. from initial testing including it speeds up models
       case aten::max_pool2d:
       case aten::max_pool3d:
+      case aten::adaptive_avg_pool2d:
         return true;
     }
 
+    if (n->kind() == aten::hardtanh) {
+      auto min_val = constant_as<double>(n->namedInput("min_val")).value();
```

Comment:
`constant_as<double>(n->namedInput("min_val")).value()` is going to throw if this value isn't a constant.  you want: `min_val = constant_as<double>(n->namedInput("min_val")).value_or(1.)` and `max_val = constant_as<double>(n->namedInput("max_val")).value_or(-1)`

Reply:
Ah, right. I didn't realize the other places in that file guard accesses with `nonConstantParameters`.